_posts/2022-05-09-ooh-na-na-where-are-my-sodium-labs/the-na-bug.Rmd

```diff
@@ -6,15 +6,15 @@ author:
   - name: Chloe Pou-Prom
```

Comment:
Unrelated to this file, but what do you think of adding a PR checklist for the blog post team for when we're publishing? (Could contain some of the same items as the writer's checklist, like ensuring the Rmd knits properly etc.)

Reply:
Yes, I think that makes a lot of sense! I think we should either update the PR template to include this checklist: https://github.com/LKS-CHART/blog/blob/main/.github/pull_request_template.md or update the ClickUp document or both

Reply:
Sounds good, I will add it to the PR template/clickup doc!